library/ns1_zone.py

```diff
@@ -7,8 +7,10 @@
 
 __metaclass__ = type
 
-
-from ansible.module_utils.ns1 import NS1ModuleBase, HAS_NS1
+try:
+    from ansible.module_utils.ns1 import NS1ModuleBase, HAS_NS1
+except ImportError:
+    from module_utils.ns1 import NS1ModuleBase, HAS_NS1 # noqa
```

Comment:
this is a 2/3 thing?

Reply:
Ooh good catch, will add a comment.  The import path changes depending on if this is being executed via Ansible or directly via pytest.